docker/setup.sh

```diff
@@ -75,20 +163,20 @@ if [[ ! -f "/client/extracted" ]]; then
     echo "client/res/maps/**" >> globs.txt
     echo "*.fdb" >> globs.txt
 
-    lunpack -g ./globs.txt /client/
+    lunpack -g ./globs.txt $CLIENT_ROOT_DIR/..
 
-    touch /client/extracted
+    touch /docker/extracted
 else
     echo "Client already extracted. Skip this step..."
     echo "If you want to force a re-extract, just delete the file called \"extracted\" in the client directory"
```

Comment:
Is this still correct? If I understand it correctly, with the changes of this PR the `extracted` and `migrated` file (see message below which needs to updated as well) are now created in the `shared_configs` volume. Therefore they won't exist in the `client` directory; unless someone used the previous Docker setup, but then the files simply have no effect.

Reply:
Oh, now I understand what you mean. Yes, then you are completely right. The message isn't up to date .

Reply:
I changed the message which should be more helpful.